src/cursor/cur_backup.c

```diff
@@ -229,6 +229,7 @@ __backup_start(
 	 */
 	__wt_writelock(session, conn->hot_backup_lock);
 	conn->hot_backup = true;
+	conn->hot_backup_list = NULL;
 	__wt_writeunlock(session, conn->hot_backup_lock);
```

Comment:
@sueloverso, can you please update the comment above to explain that we're relying on the schema lock to prevent `drop` operations while building the list?